src/fdopendir.c

```diff
@@ -30,6 +30,18 @@ int best_fchdir(int dirfd);
 
 #define PROTECT_ERRNO(what)  ({ int __err = (errno); what; errno = __err; })
 
+/*
+  https://www.freebsd.org/cgi/man.cgi?query=fdopendir&sektion=3
+
+  Upon successful return from fdopendir(), the file descriptor is under the
+  control of the system, and if any attempt is made to close the file
+  descriptor, or to modify the state of the associated description other
+  than by means of closedir(), readdir(), readdir_r(), or rewinddir(), the
+  behavior is undefined.  Upon calling closedir() the file descriptor is
+  closed.  The FD_CLOEXEC flag is set on the file descriptor by a
+  successful call to fdopendir().
+*/
+
```

Comment:
Please also note https://linux.die.net/man/3/fdopendir  :
```
After a successful call to fdopendir(), fd is used internally by the implementation, and should not otherwise be used by the application.

The fdopendir() function leaves the setting of the close-on-exec flag unchanged for the file descriptor, fd. POSIX.1-200x leaves it unspecified whether a successful call to fdopendir() will set the close-on-exec flag for the file descriptor, fd.
```

Reply:
I replaced the documentation with the two man page links instead to avoid possible licensing issues with inlining the text.

Reply:
nice!